oauth2/oauth2-resource-server/src/main/java/org/springframework/security/oauth2/resourceserver/web/BearerTokenAuthenticationEntryPoint.java

```diff
@@ -60,18 +60,18 @@ public void commence(HttpServletRequest request, HttpServletResponse response,
 		if (authException instanceof BearerTokenAuthenticationException) {
 			BearerTokenError error = ((BearerTokenAuthenticationException) authException).getError();
 			httpStatus = error.getHttpStatus();
-			authParamAttributes.put("error", error.getErrorCode());
+			authParamAttributes.put(OAuth2ParameterNames.ERROR, error.getErrorCode());
 			String description = error.getDescription();
 			if (description != null) {
-				authParamAttributes.put("error_description", description);
+				authParamAttributes.put(OAuth2ParameterNames.ERROR_DESCRIPTION, description);
```

Comment:
I'm not totally convinced that we should reuse `OAuth2ParameterNames` here as I suggested. Let's leave it out for now.

Reply:
okay, I'll take it out for now. By the way, what makes you hesitant? AFAICT, this entrypoint is OAuth2-specific (e.g. tied to RFC 6750).

Reply:
If you look at the `OAuth2ParameterNames` javadoc and associated link to the [11.2 OAuth Parameters Registry](https://tools.ietf.org/html/rfc6749#section-11.2), these are parameters that are defined and used by the Authorization Endpoint and Token Endpoint. Hence my hesitation as we're dealing with Resource Server here. 

However, the reason I originally suggested is that an OAuth 2.0 Error is applicable across the board with Client, Authorization Server and Resource Server so maybe it makes sense? At the moment I'm ok with not reusing the 3 parameter names here.